tutorials/hess.ipynb

```diff
@@ -72,12 +69,64 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "data_store = DataStore.from_file(\n",
-    "    \"$GAMMAPY_DATA/hess-dl3-dr1/hess-dl3-dr3-with-background.fits.gz\"\n",
-    ")\n",
-    "mask = data_store.obs_table[\"TARGET_NAME\"] == \"Crab\"\n",
-    "obs_table = data_store.obs_table[mask]\n",
-    "observations = data_store.get_observations(obs_table[\"OBS_ID\"])"
+    "config = \"\"\"\n",
+    "general:\n",
+    "    logging:\n",
+    "        level: INFO\n",
+    "    outdir: .\n",
+    "\n",
+    "observations:\n",
+    "    datastore: $GAMMAPY_DATA/hess-dl3-dr1/hess-dl3-dr3-with-background.fits.gz\n",
+    "    filters:\n",
+    "        - filter_type: par_value\n",
+    "          value_param: Crab\n",
+    "          variable: TARGET_NAME\n",
+    "\n",
+    "reduction:\n",
+    "    dataset-type: MapDataset\n",
+    "    stack-datasets: true\n",
+    "    offset-max: 2.5 deg\n",
+    "    psf-kernel-radius: 0.3 deg\n",
+    "    geom:\n",
+    "        skydir: [83.633, 22.014]\n",
+    "        width: [5, 5]\n",
+    "        binsz: 0.02\n",
+    "        coordsys: CEL\n",
+    "        proj: TAN\n",
+    "        axes:\n",
+    "          - name: energy\n",
+    "            hi_bnd: 30\n",
+    "            lo_bnd: 1\n",
+    "            nbin: 5\n",
+    "            interp: log\n",
+    "            node_type: edges\n",
+    "            unit: TeV\n",
+    "    geom-irf:\n",
```

Comment:
Suggest to remove `geom-irf` for now, keep it simple.

Reply:
An issue similar as the one below, we have to ensure that definition of IRF is correct given the IRF stored values and the choice of the user reco energy bins.

Reply:
@registerrier What do you exactly mean by this? The binning for the `PSFKernel` and `EnergyDispersion` are deferred from the true and reconstructed geoms internally. I think those are chose correctly...

My much preferred solution for this would be to go to a simpler parametrisation for now, as I've proposed a few times:

`MapDataset.create(geom, energy_true_axis, rad_axis, migra_axis, bins_irf)` and later add a `margin` argument to account for spill over.

Reply:
I simply meant that `energy_true_axis` has to be chosen in a way that is consistent with what is in the IRF and with the energy reco binning. e.g. if `e_reco` starts at 300 GeV, `e_true` should start sufficiently below, typically 100 GeV.

Reply:
Thanks for the explanation, that makes sense...I guess it's somehow a more general problem, that we don't handle in the code yet. I think we don't check e.g. `e_true` and `e_reco` in the `SpectrumExtraction` either. But I agree it would be simpler to remove the `geom-irf` in the config file for this analysis...

Reply:
Note that Fermi ST and ctools don't expose it like this to end-users.
One can choose an energy binning for counts (`gtbin`), and for exposure (`gtexpcube2`), but otherwise things are always chosen and handled internally.
I think the requirement on the user is to (a) choose counts and exposure with energy binnings that aren't too large (so that PL approximations for interpolation and integration are fine) and such that exposure covers the counts range.
If you open up more options and combinations in the high-level interface you will get more usage error.
I don't know how Fermi ST chooses geoms e.g. for PSF and EDISP internally.

Reply:
I agree with you @cdeil. I think we should expose `geom` and `energy_true_axis` for now, which is sufficient for 98% of the users. Later we can maybe add  `rad_axis`, `migra_axis` and `binsz_irf` for consistency and "power-users". The alignment of the different maps and things like margin for the exposure and IRF maps should be handled internally.

Reply:
I removed `geom-irf` for now...